include/odp/api/spec/schedule.h

```diff
@@ -371,6 +371,30 @@ void odp_schedule_order_unlock(uint32_t lock_index);
  */
 void odp_schedule_order_unlock_lock(uint32_t unlock_index, uint32_t lock_index);
 
+/** Asynchronous ordered context lock
+ * Request an ordered context lock to be acquired. Starts to acquire an ordered
+ * context lock does not wait until the lock has been acquired. Application can
```

Comment:
Suggesting re-phrase the sentence: "Starts an ordered context lock acquire operation, but does not wait until..."